aquacomputer_d5next.c

```diff
@@ -362,6 +375,13 @@ static const char *const label_leakshield_fan_speed[] = {
 	"Reservoir Filled [ml]",
 };
 
+/* Labels for AquastreamXT */
+static const char *const label_temp_sensors_aquastreamxt[] = {
+	"Fan ctrl temp",
```

Comment:
Just for my reference, if I'm understanding correctly this is the temp that the pump can use to control the optionally attached fans?

Reply:
No, this is the temp of the IC controlling the fan voltage.